web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -217,6 +212,7 @@ details.captcha > div {
    padding: 12px 0px;
 }
 
+/*
```

Comment:
We commented out loads of stuff. Lets just delete it as soon as we know we don't need it.

Reply:
Done.